lib/SparkPost/SparkPost.php

```diff
@@ -45,6 +46,7 @@ public function __construct($httpAdapter, $settingsConfig) {
     $this->setHttpAdapter($httpAdapter);
 
     $this->transmission = new Transmission($this);
+    $this->messageEvent = new MessageEvent($this);
```

Comment:
Same here. We should probably call the property `messageEvents` since users will be interacting with collections.